test/EntityFramework/FunctionalTests/Query/SqlCeCanonicalFunctionsTests.cs

```diff
@@ -514,7 +514,7 @@ public void CanonicalFunction_truncatetime_translated_properly_to_sql_function()
             using (var context = GetArubaCeContext())
             {
                 var query = context.CreateQuery<DateTime>(@"SELECT VALUE Edm.TruncateTime(A.c5_datetime) FROM ArubaCeContext.AllTypes AS A");
-                Assert.Contains("CONVERT", query.ToTraceString().ToUpperInvariant());
+                Assert.Contains("DATEADD(D, DATEDIFF(D,", query.ToTraceString().ToUpperInvariant());
```

Comment:
We only have canonical function tests for SQL CE? :man_facepalming: Glad we have coverage for this in https://github.com/aspnet/EntityFramework6/blob/master/test/EntityFramework/FunctionalTests/ProductivityApi/DbFunctionScenarios.cs.